pkg/core/blockchain.go

```diff
@@ -686,6 +724,7 @@ func (bc *Blockchain) GetStateModule() blockchainer.StateRoot {
 // transactions with all appropriate side-effects and updates Blockchain state.
 // This is the only way to change Blockchain state.
 func (bc *Blockchain) storeBlock(block *block.Block, txpool *mempool.Pool) error {
+	bc.log.Info("adding block", zap.Uint32("index", block.Index))
```

Comment:
Debug

Reply:
We don't need this log, it was saved by mistake, so I removed it.